omnibus-mainnet.toml

```
@@ -7,20 +7,24 @@ include = [
     "tomls/core.toml",
     "tomls/permissions.toml",
     "tomls/collaterals/snx.toml",
+    "tomls/markets/legacy.toml",
     # "tomls/oracles/eth.toml",
     # "tomls/collaterals/weth.toml",
     "tomls/pools/spartan-council.toml",
     "tomls/permit-all-transferCrossChain.toml",
 ]
 
 [setting.snx_package]
-defaultValue = "synthetix:3.3.5"
+defaultValue = "synthetix:3.3.15"
```

the latest is 18 now
```suggestion
defaultValue = "synthetix:3.3.18"
```